### PR TITLE
add validation chk to disable the submit button. fix missing title

### PR DESCRIPTION
**Pull request by rox163**
#MAJOR#

# CHANGELOG
* Adding validation trigger on submit button click
* Shows validation error count
* Fix missing title

### CODE REVIEW

File: app/templates/components/frost-modal-input.hbs [hunk 1]
```diff
@@ -11,20 +11,16 @@
   {{/block-slot}}
 
   {{#block-slot slot 'header'}}
-    {{#yield-slot 'header'}}
-      {{yield}}
+    {{#if titleComponent}}
+      {{component titleComponent}}
     {{else}}
```
Comment (rox163) [inline app/templates/components/frost-modal-input.hbs]: else yield doesnt work. it wasnt supposed to go in :(

**Comment by vesper2000** [comment]
👍

**Comment by sglanzer** [comment]
Delaying on this as discussed until the validate action closure is wired in

**Comment by sglanzer** [comment]
Approved

**Pull request merged**